src/styles/_catalog.scss

```diff
@@ -1,8 +1,8 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
```

Comment:
there is too mutch space between your cards, probably because your columns stretches, try to replace "minmax(200px, 1fr)" just to "200px"  and in every media too